rules/windows/powershell/powershell_CL_Invocation_LOLScript.yml

```diff
@@ -0,0 +1,35 @@
+title: Execution via CL_Invocation.ps1
+id: 4cd29327-685a-460e-9dac-c3ab96e549dc
+description: Detects Execution via SyncInvoke in CL_Invocation.ps1 module
+status: experimental
+author: oscd.community, Natalia Shornikova
+date: 2020/10/14
+references:
+    - https://github.com/LOLBAS-Project/LOLBAS/blob/master/yml/OSScripts/Cl_invocation.yml
+    - https://twitter.com/bohops/status/948061991012327424
+tags:
+    - attack.defense_evasion
+    - attack.t1216
+logsource:
+    product: windows
+    service: powershell
+detection:
+    selection:
+        EventID: 4104
+        ScriptBlockText|contains|all:
+          - 'CL_Invocation.ps1'
+          - 'SyncInvoke'
+    selection2:
+        EventID: 4104
+        ScriptBlockText|contains:
+          - 'CL_Invocation.ps1'
+          - 'SyncInvoke'
+    timeframe: 1m
+    condition: 
+        - selection
+      # PS > Import-Module c:\Windows\diagnostics\system\Audio\CL_Invocation.ps1; SyncInvoke c:\Evil.exe
+        - selection2 | count(ScriptBlockText) by Computer > 2
```

Comment:
I am not sure is this way of declaring the condition is correct. Could you please rewrite it in a common way?

Reply:
I used [this rule](https://github.com/Neo23x0/sigma/blob/82cae6d63c9c2f6d3e86c57e11497d86279b9f95/rules/windows/builtin/win_susp_failed_logons_single_source.yml) as an example. And for this rule, the sigmac produces only the first condition (selection1). I checked this on my rule, I swapped the conditions so that my selection2 is now the first, then the sigmac produces:
Splunk:
(source="WinEventLog:Microsoft-Windows-PowerShell/Operational" EventCode="4104" (ScriptBlockText="*CL_Invocation.ps1*" OR ScriptBlockText="*SyncInvoke*")) | eventstats dc(ScriptBlockText) as val by Computer | search val > 2

So it seems sigmac converts only the first condition it finds

Reply:
I believe that you can use `or` operator here (:

Reply:
You mean:
 -selection or ( selection2 | count(ScriptBlockText) by Computer > 2)

or just leave only this selection:

EventID: 4104
ScriptBlockText|contains:
-'CL_Invocation.ps1'
-'SyncInvoke'
condition: 
-selection
 ?

Reply:
Hey! I meant the first option (;

Reply:
There was an [error](https://github.com/Neo23x0/sigma/pull/1184/checks?sha=a6f00d6acc57ec9f41c6dd9f5fe86778aeda47e7) when changed it to just `or` operator :( So I returned it as it was for now

Reply:
Hello @nsaddler! What if we will remove the second selection?
Will it affect the detection rate somehow? 
Maybe it makes sense to move it into a separate rule? 
I don't see a way to keep both conditions in the same rule. Tried to do a few tricks, but all failed.

Reply:
Hello @nsaddler! Would you like to proceed with working on this rule?

Reply:
Hello, sorry for delay!
I would like to split rule into two